os/drivers/pm/pm.c

```diff
@@ -119,6 +119,13 @@ static int pm_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 			ret = pm_timedstay(((pm_suspend_arg_t *)arg)->state, ((pm_suspend_arg_t *)arg)->timer_interval);
 		}
 		break;
+	case PMIOC_STAYCOUNT:
+		if ((enum pm_state_e)arg >= PM_NORMAL && (enum pm_state_e)arg <= PM_SLEEP) {
+			ret = pm_staycount(PM_IDLE_DOMAIN, (enum pm_state_e)arg);
+		} else {
+			pmvdbg("Please input valid PM state \n");
```

Comment:
For fail case, maybe pmdbg?

Reply:
why do you read using driver, not procfs?
if you want read in here, please remove procfs